examples/cpp/comm/comm_dll/using_comm_dll.cpp

```diff
@@ -82,7 +82,7 @@ int main()
 
     // Exersize Comm facilities
 
-    bond::comm::SocketAddress loopback("127.0.0.1", 25188);
+    bond::comm::SocketAddress loopback("127.0.0.1", EXAMPLE_PORT_1);
```

Comment:
If we're talking specifically about examples, they only need to be given unique ports like this because we use our examples as tests. If this became a problem because we wanted to run them elsewhere, I'd rather strike at the root cause - get them out of the test framework entirely and let them use the default port.